masterlist.yaml

```diff
@@ -37,6 +37,16 @@ common:
         text: "Du ser ud til at bruge %1%, men du bruger ikke “%1%”-lappen for dette plugin: %2%"
       - lang: ja
         text: "%1%が使用されていますが、このプラグイン用の%1%のパッチが有効になっていません: %2%"
+  - &hasPatchIncluded
+    type: warn
+    content:
+      - lang: en
+        text: "A patch is included with the %2% installer, please make sure you have installed the %1% patch and have it activated"
```

Comment:
I've seen a good few cases of people installing a mod which had a patch option for another mods. And them later reporting a conflict, when they had just installed in the wrong order.